velox/common/caching/AsyncDataCache.h

```diff
@@ -295,7 +295,7 @@ class AsyncDataCacheEntry {
   // SSD. The exact file and offset are needed to include uses in RAM
   // to uses on SSD. Failing this, we could have the hottest data first in
   // line for eviction from SSD.
-  tsan_atomic<SsdFile * FOLLY_NULLABLE> ssdFile_{nullptr};
+  tsan_atomic<SsdFile * FOLLY_NULLABLE> FOLLY_NULLABLE ssdFile_{nullptr};
```

Comment:
Wonder why the first `FOLLY_NULLABLE` is not enough

Reply:
`tsan_atomic<SsdFile *> FOLLY_NULLABLE ssdFile_{nullptr};`?

Reply:
Without the second FOLLY_NULLABLE I added, it will fail the build, see detailed log in https://github.com/facebookincubator/velox/issues/3407. I guess it's a M1 related issue.  

Updated with @duanmeng 's feedback.